shoulder_pan_joint: 0.0
shoulder_lift_joint: -1.2
elbow_joint: -2.3
wrist_1_joint: -1.2
wrist_2_joint: 1.57
wrist_3_joint: 2.8